publish-mqtt/src/main.rs

```diff
@@ -208,27 +249,34 @@ async fn requests(mut homie: HomieDevice) -> Result<(), Box<dyn Error>> {
                         Some(path) => path,
                         None => continue,
                     };
-                    let device = BluetoothDevice::new(bt_session, device_path.to_string());
-
-                    if let Some(readings) = decode_value(&value) {
-                        let (node_id, name) = node_id_name_for_sensor(&device, &sensor_names)?;
-                        println!("{} {} ({})", device.get_id(), readings, name);
-
-                        homie
-                            .publish_value(
-                                &node_id,
-                                "temperature",
-                                format!("{:.2}", readings.temperature),
-                            )
-                            .await?;
-                        homie
-                            .publish_value(&node_id, "humidity", readings.humidity)
-                            .await?;
-                        homie
-                            .publish_value(&node_id, "battery", readings.battery_percent)
-                            .await?;
+                    if let Some(sensor) = sensors_connected
+                        .iter_mut()
+                        .find(|s| s.device_path == device_path)
+                    {
+                        sensor.last_update_timestamp = Instant::now();
+                        if let Some(readings) = decode_value(&value) {
+                            println!("{} {} ({})", sensor.mac_address(), readings, sensor.name());
```

Comment:
```suggestion
                        if let Some(readings) = decode_value(&value) {
                            println!("{} {} ({})", sensor.mac_address(), readings, sensor.name());
                            sensor.last_update_timestamp = Instant::now();
```

If we get a reading that's bogus, does it really count as a reading?

Reply:
As far as being fairly confident that we are still connected, I think it does count.